### PR TITLE
Added a new page about std::pair.

### PR DESCRIPTION
**Pull request by sbofgayschool**
Quite useful on some occasions, like discretizing input and processing related data.

### CODE REVIEW

**Comment by sbofgayschool** [comment]
有大佬愿意review一下本蒟蒻的这个pr么～？🥺🥺

File: docs/lang/csl/pair.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+p1.second = 2.0;
+```
+
+还可以使用 `std::make_pair` 函数。该函数接受两个变量，并返回由这两个变量组成的 `pair`。
+
+```cpp
+pair<int, double> p2 = make_pair(1, 2.0);
+```
+
+大佬们通常喜欢使用宏定义 `#define make_pair mp`，将有些冗长的 `make_pair` 化简为 `mp`。
```
Comment (Enter-tainer) [inline docs/lang/csl/pair.md:43]:
```suggestion
一种常用的方法是使用宏定义 `#define make_pair mp`，将有些冗长的 `make_pair` 化简为 `mp`。
```

**Review by Enter-tainer — APPROVED** [review]
thx!!

**Review by Enter-tainer — APPROVED** [review]

**Review by Xeonacid — APPROVED** [review]
thx

**Pull request merged**